web/templates/stream_settings/stream_types.hbs

```diff
@@ -82,7 +82,7 @@
 
                     <select name="stream_message_retention_setting"
                       class="stream_message_retention_setting prop-element settings_select bootstrap-focus-style"
-                      id="id_message_retention_days"
+                      id="{{prefix}}message_retention_days"
```

Comment:
`id_new_message_retention_days` ID for the stream creation form seems weird but other names already have `stream_` in them and this does not. We do not use this ID directly anywhere so should be fine.